db/seeds.rb

```diff
@@ -68,14 +68,42 @@
   })
 end
 
+Parent.destroy_all
 puts "Creating Parents..."
 
+Kid.destroy_all
+puts "Creating Kids..."
```

Comment:
Instead, maybe do `puts "Creating Parents and Kids..."` as it's more explanatory.